### ISSUE TITLE
It's not clear how a "token" should be generated

### ISSUE THREAD

**Issue by almet** [Kinto/kinto#976]
Today, we've got feedback from @simon-previdente where he was stating that Kinto was great but that he lost 2 hours on authentication before understanding that any token was valid.

The information is [already available in the FAQ](http://kinto.readthedocs.io/en/stable/faq.html#how-does-kinto-authenticate-users) but apparently is not as easy to find as it should.

Maybe we should add an "authentication" section in the "concepts" page?

**Comment by leplatrem** [comment]
Side note: we'll probably drop that weird `basicauth` authentication once #795 will land.

**Comment by almet** [comment]
Thanks for the info. depending on when we plan to ship #795, it might or might not make sense to document this.

**Comment by enguerran** [comment]
I recognize that it is not obvious as usually you need to manage user creation before using it. I don't know another application in which the authentication is made that way.

So it may be a good idea to insist and find a beautiful metaphor to illustrate. If I get an idea, I'll let you know.

**Comment by leplatrem** [comment]
We'll drop basic auth in #1736 

Closing as won't fix :)
